extensions/amp-lightbox-viewer/0.1/amp-lightbox-viewer.js

```diff
@@ -207,6 +219,25 @@ export class AmpLightboxViewer extends AMP.BaseElement {
   }
 
   /**
+   * Toggle the overflow state of description box
+   * @private
+   */
+  toggleDescriptionOverflow_() {
+    if (this.descriptionBox_.classList.contains('standard')) {
```

Comment:
use vsyn and do the `scrollHeight` `clientHeight` reads in the measure block and then mutate